client/components/createTM/tab-choose.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
```

Comment:
lets rename this file to `tab-select-game.js`.

Rename the component to `TabSelectGame`.